src/Agent/NewRelic/Profiler/MethodRewriter/ISystemCalls.h

```diff
@@ -7,14 +7,73 @@
 #include <string>
 #include <set>
 #include "../Common/xplat.h"
+#include "../Logging/DefaultFileLogLocation.h"
 
 namespace NewRelic { namespace Profiler { namespace MethodRewriter {
-    struct ISystemCalls
+    struct ISystemCalls : Logger::IFileDestinationSystemCalls
     {
         virtual std::unique_ptr<xstring_t> TryGetEnvironmentVariable(const xstring_t& variableName) = 0;
-        virtual xstring_t GetNewRelicHomePath() = 0;
-        virtual xstring_t GetNewRelicInstallPath() = 0;
         virtual bool FileExists(const xstring_t& filePath) = 0;
+
```

Comment:
Moving these to the base class/interface is a great way to avoid repeating ourselves. 👍